ads/google/page-level-experiments.js

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
```

Comment:
I imagine it would be useful outside the google scope and nothing in page-level-experiments.js is specific to google (or even ads) now so happy to move if @lannka is happy with that?